files/zh-cn/web/mathml/element/msub/index.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: <msub>
 slug: Web/MathML/Element/msub
+l10n:
+  sourceCommit: 4d5e2c11f4b8cc32e54d2527d9576ed26ced9458
```

Comment:
Please re-translate line 10 to make it consistent with the rest of this sections's API documentation.